src/frontend/src/lib/components/send/SendContacts.svelte

```diff
@@ -1,11 +1,46 @@
 <script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { createEventDispatcher } from 'svelte';
+	import SendContact from '$lib/components/send/SendContact.svelte';
 	import EmptyState from '$lib/components/ui/EmptyState.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
+	import type { ContactUi } from '$lib/types/contact';
+	import type { NetworkContacts } from '$lib/types/contacts';
 
-	// TODO: display filtered contacts in this component
+	interface Props {
+		destination: string;
+		networkContacts?: NetworkContacts;
+		selectedContact?: ContactUi;
+	}
+
+	let {
+		networkContacts,
+		selectedContact = $bindable(),
+		destination = $bindable()
+	}: Props = $props();
+
+	const dispatch = createEventDispatcher();
 </script>
 
-<EmptyState
-	title={$i18n.send.text.contacts_empty_state_title}
-	description={$i18n.send.text.contacts_empty_state_description}
-/>
+{#if nonNullish(networkContacts) && Object.keys(networkContacts).length > 0}
+	<div class="flex flex-col overflow-y-hidden sm:max-h-[13.5rem]">
+		<ul class="list-none overflow-y-auto overscroll-contain">
+			{#each Object.keys(networkContacts) as address, index (index)}
+				<SendContact
+					contact={networkContacts[address]}
+					{address}
+					on:click={() => {
```

Comment:
Should this not be a prop for svelte 5 components?

```suggestion
					onClick={() => {
```